packages/mixer/src/Mixer.tsx

```diff
@@ -1,15 +1,75 @@
 import { Deposit, Withdraw } from '@webb-dapp/mixer/components';
 import { MixerTabs } from '@webb-dapp/ui-components/Tabs/MixerTabs';
-import React from 'react';
+import { CheckBox, DropdownMenu, Input } from '@webb-dapp/webb-ui-components';
+import {
+  Coin,
+  Filter,
+  GithubFill,
+  Graph,
+  Search,
+  TelegramFill,
+  TwitterFill,
+} from '@webb-dapp/webb-ui-components/icons';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
 const MixerWrapper = styled.div``;
 type MixerProps = {};
 
 export const Mixer: React.FC<MixerProps> = () => {
+  const dropdownOptions = [
+    { value: 'Github', icon: <GithubFill /> },
+    { value: 'Telegram', icon: <TelegramFill /> },
+    { value: 'Twitter', icon: <TwitterFill /> },
+  ];
```

Comment:
Could be moved to be constant outside the component  or `useState` in order for it to be instantiated once

Reply:
@AtelyPham  please fix the merge conflicts